includes/wikia/DefaultSettings.php

```diff
@@ -1773,7 +1773,7 @@
  * A central regex string for use in domain checking, so we can easily
  * update/add/change domains in the future
  */
-$wgWikiaBaseDomainRegex = '((wikia|fandom)\\.com|(wikia|fandom)-dev\\.(com|us|pl))';
+$wgWikiaBaseDomainRegex = '((wikia|fandom)\\.(com|org)|(wikia|fandom)-dev\\.(com|us|pl))';
```

Comment:
I'm not sure if we actually own `fandom.org`, so, given this can be used for validating inputs, I think it would be safer to separate this (we could have `wikia\\.(com|org)|fandom\\.com` for example).